collections/business/general-notifications/event-general-notifications/pp-check-exceptions-created/event-general-notifications.json

```diff
@@ -16,6 +16,7 @@
             "channel": "in-app-notification",
```

Comment:
@amorimluzia is it necessary to bump backbase-bom version?
https://github.com/Backbase/engagements-data/blob/master/pom.xml#L15

Reply:
yes, it is. Please @denyso-bb upgrade the backbase-bom to 2024.04

Reply:
yes, it is. Please @denyso-bb upgrade the backbase-bom to 2024.04